templates/base.html.twig

```diff
@@ -6,7 +6,7 @@
 {% set default_sharer_width = 1905 %}
 {% set default_sharer_height = 1000 %}
 
-{% set user_is_adherent = is_granted('ROLE_ADHERENT') %}
+{% set user_is_user = is_granted('ROLE_USER') %}
```

Comment:
`user_is_adherent`? the current naming looks weird.

Reply:
Now you can be registered in en-marche without being an adherent. The naming is weird.